toolbox/org.lamport.tla.toolbox.doc/html/model/tlc-options-page.html

```diff
@@ -96,6 +101,8 @@ <h3><a  name="model-mode">Model-checking mode</a></h3>
 queue, so it does not produce any
  <i>Diameter</i> or <i>Queue size</i>
  statistics.
+Additionally, <em>Depth-first</em> option doesn't currently
```

Comment:
The confusion typically comes from the fact that TLC does *not* plain depth-first but depth-first iterative deepening (DFID).  We should consistently call it by its full name.

Reply:
Thanks for the insight! I've updated the paragraph.